### DISCUSSION TITLE
Schedule tasks

### DISCUSSION THREAD

**Discussion by sgaseretto** [GenieFramework/Genie.jl#500] [Q&A] [answered]
I see in the readme that you can define tasks to run on a schedule with Genie, but is there a way to set them up and schedule them with Genie? Couldn't find something explaining how to do it in the docs

**Comment by AbhimanyuAryan** [comment, marked as answer]
@sgaseretto in Genie tasks are basically julia scripts that you can run at any given time. But Genie doesn't yet offer task scheduling logic for tasks out of the box(not that I am aware of) 

```julia
$ bin/runtask TaskName
```

API here: https://www.genieframework.com/docs/api/toolbox.html

You can look at example tasks here: https://github.com/GenieFramework/PkgVizBoard/tree/master/tasks

From the above tasks if you pick `S3DBTask` it is responsible to dump data in db. You can also hook this with a cron job 

I thought of implementing a [crontab](https://github.com/kraiz/django-crontab) as Genie Plugin(in free time) but due to some other priorities didn't made any progress. If you have any other idea let us know. Would be happy to review any contributions made :)

**Comment by sgaseretto** [comment]
Thanks a lot for the rapid response @AbhimanyuAryan ! I thought that scheduling was integrated, but now I understand what are Genie tasks meant for. Searching for something to schedule tasks, I found [JobSchedulers.jl](https://github.com/cihga39871/JobSchedulers.jl). I'll experiment with it and how to add it as a plugin to Genie once I get more free time from my job!

**Reply by AbhimanyuAryan** [reply]
Looking forward 😍

**Reply by essenciary** [reply]
Great finding re the scheduler package!
